mne/viz/backends/_qt.py

```diff
@@ -466,6 +498,25 @@ class _Renderer(_PyVistaRenderer, _QtDock, _QtToolBar, _QtMenuBar,
     pass
 
 
+def _detect_theme():
+    try:
+        import darkdetect
+    except ModuleNotFoundError:
+        logger.info('For automatic Dark-Mode-Detection '
+                    '"darkdetect" has to be installed!'
+                    'You can install it with'
+                    ' `pip install qdarkdetect`')
+        theme = 'light'
+    else:
+        detected_theme = darkdetect.theme()
+        if detected_theme is not None:
+            theme = detected_theme.lower()
+        else:
+            theme = 'light'
+
+    return theme
```

Comment:
One problem with this flow is that, because `'auto'` is the default (which I think is the right choice), anybody without `darkdetect` who uses the `Brain` class  / `stc.plot` for anything without explicitly setting `theme` will get this `logger.info` message saying that no dark mode detection can be done. I'd rather make things simpler and do:
```suggestion
def _detect_theme():
    try:
        import darkdetect
        return darkdetect.theme().lower()
    except Exception:
        return 'light'
```
We already document in the parameter that they should have `darkdetect` for things to work properly so I don't think we need to `logger` them about it. This code is also robust to any issue with `darkdetect` we might encounter (at the cost of failing silently when it doesn't work, which I think is an acceptable tradeoff for non-essential aesthetic UI functionality)